argocd/base/servicemonitors/argocd-metrics.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: argocd-metrics
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: argocd-metrics
+  endpoints:
+  - port: metrics
+  namespaceSelector:
+    matchNames:
+      - opf-monitoring
```

Comment:
argocd is deployed in namespace: `argocd`

Reply:
fixed, thanks!